balboa-admin/src/main/resources/application.conf

```diff
@@ -0,0 +1,49 @@
+# Logging
+log4j {
+  logger {
+    com.socrata.balboa {
+      0: ${?BALBOA_LOGLEVEL}
+      1: ${?BALBOA_LOGDEST}
+    }
+    net.spy.memcached = ${?NETSPY_MEMCACHED}
+  }
+
+  # Console configuration (used in development)
+  appender.console.class = ${?LOGGER_CLASS}
+  appender.console.props {
+    layout.class = ${?LOGGER_LAYOUT_CLASS}
+    layout.props.ConversionPattern = ${?LOGGER_CONVERSION_PATTERN}
+  }
+}
+
+balboa {
+  # The levels at which to summarize. After you start summarizing, you should not
+  # ever change these existing. If you do, some tiers of summarization will contain
+  # inconsistent data with the rest.
+  summaries = ${?BALBOA_SUMMARY_LEVELS}
+
+  # Period of serialization to use. After you start summarizing, this should never
+  # change or things will break really bad. Some datastores (e.g. SQL) use their
+  # own serialization and ignore this value.
+  serializer = ${?BALBOA_SERIALIZER}
+
+  # Period of data store to use.
+  datastore = ${?BALBOA_DATASTORE}
+}
+
+cassandra {
+  # Cassandra servers. Servers should be separated by a comma.
```

Comment:
No, I missed it because it wasn't in the old config.properties file. It should be added, though. Good catch.